src/shared/lib/time.ts

```diff
@@ -21,7 +23,11 @@ function formatXX(item: number) {
   return item < 10 ? '0' + item : item;
 }
 
-export function format(timestamp: Date | string) {
+export function getUTCFormat(date: Date) {
+  return format(date, `MMM-dd-yyyy hh:mm:ss a 'UTC'`);
+}
```

Comment:
```suggestion
export function formatUTC(date: Date) {
  return format(date, `MMM-dd-yyyy hh:mm:ss a 'UTC'`);
}
```

Please add a simple unit test whenever there is a chance to test something not by looking at the browser page.